pkg/cmd/server/api/helpers.go

```diff
@@ -17,6 +17,85 @@ import (
 	cmdutil "github.com/openshift/origin/pkg/cmd/util"
 )
 
+var (
+	openShiftFeatureNames map[string]string
+)
+
+func init() {
+	openShiftFeatureNames = make(map[string]string, len(KnownOpenShiftFeatures))
+	openShiftFeatureNames["builder"] = "Builder"
```

Comment:
Why not just use the pretty names to begin with?  They're stored in a proper slice, so parsing isn't a problem.  Translation?  I'd say there's a good argument that these are proper nouns.